### PR TITLE
Fix FS Dispose Error

### PR DESCRIPTION
**Pull request by Ashuaidehao**

### CODE REVIEW

**Review by shargon — APPROVED** [review]

**Comment by erikzhang** [comment]
There is no difference.

**Review by superboyiii — APPROVED** [review]
It solved FS issue in Oracle request.
```
neo> [09:53:56.891] Process oracle request txid: 0x8fcb411287c111bbbf957726903620976d1d02a075d9f271c855577118ea0ff0, url: neofs:CMjo8ZrUtgYGenu4TXgK8mp446fqgnSh5qyZCiQhBeGr/5zvXf4n45cqgB2sqvJku9WKvGhgoh867oxNijqXg5g4E
[09:53:56.895] Request: neofs:CMjo8ZrUtgYGenu4TXgK8mp446fqgnSh5qyZCiQhBeGr/5zvXf4n45cqgB2sqvJku9WKvGhgoh867oxNijqXg5g4E
get object, deadline=, context=False
[09:53:57.025] NeoFS result: error, False Grpc.Core.RpcException: Status(StatusCode="Cancelled", Detail="gRPC call disposed.")
   at Grpc.Net.Client.Internal.HttpContentClientStreamReader`2.MoveNextCore(CancellationToken cancellationToken)
   at Neo.FileStorage.API.Client.Client.GetObject(GetRequest request, Nullable`1 deadline, CancellationToken context)
   at Neo.FileStorage.API.Client.Client.GetObject(Address address, Boolean raw, CallOptions options, CancellationToken context)
   at Neo.Plugins.OracleNeoFSProtocol.GetPayloadAsync(Client client, Address addr, CancellationToken cancellation)
   at Neo.Plugins.OracleNeoFSProtocol.ProcessAsync(Uri uri, CancellationToken cancellation)
```

File: src/OracleService/Protocols/OracleNeoFSProtocol.cs [hunk 1]
```diff
@@ -75,14 +75,14 @@ private Task<string> GetAsync(Uri uri, string host, CancellationToken cancellati
             var tokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellation);
             tokenSource.CancelAfter(Settings.Default.NeoFS.Timeout);
             if (ps.Length == 2)
-                return GetPayloadAsync(client, objectAddr, tokenSource.Token);
-            return ps[2] switch
+                return await GetPayloadAsync(client, objectAddr, tokenSource.Token);
```
Comment (Ashuaidehao) [inline src/OracleService/Protocols/OracleNeoFSProtocol.cs:78]: Missing `await` will let client dispose immediately here, which we want it dispose after function `GetPayloadAsync` complete.

Comment (erikzhang) [reply src/OracleService/Protocols/OracleNeoFSProtocol.cs:78]: I don't think so. I think something must be wrong elsewhere in the code.

Comment (Ashuaidehao) [reply src/OracleService/Protocols/OracleNeoFSProtocol.cs:78]: I created a script to show this small difference, https://dotnetfiddle.net/YiEMA7

Comment (superboyiii) [reply src/OracleService/Protocols/OracleNeoFSProtocol.cs:78]: > I don't think so. I think something must be wrong elsewhere in the code.

That's true. We've tested it.

**Review by erikzhang — APPROVED** [review]

**Pull request merged**